### DISCUSSION TITLE
Cameras - How do I add to bubble card?

### DISCUSSION THREAD

**Discussion by landocarlesien** [Clooos/Bubble-Card#911] [Questions about config, custom styles and templates] [answered]
Hey, I have a bunch of cameras I want to display within a bubble card. After I create the "Camera" button, how do I add them to the bubble card? Thanks.

**Comment by Clooos** [comment, marked as answer]
Hi! You need to create a pop-up for that, but watch this video first, it explains everything you need.

https://www.youtube.com/watch?v=7mOV7BfWoFc
